extensions/amp-story/1.0/amp-story-desktop-one-panel.css

```diff
@@ -18,4 +18,22 @@
 
 
 .i-amphtml-story-desktop-one-panel {
+  --large-desktop-ratio: 69 / 116;
```

Comment:
please prefix with `--i-amphtml-story` so publishers can't override it. Plz also add something in the name to say that this is the ratio of the story-page